packages/@o3r/components/package.json

```diff
@@ -12,6 +12,9 @@
     "otter-cms"
   ],
   "exports": {
+    "./package.json": {
```

Comment:
It is not conflicting with the ngPackagr process adding it? It does not print warning/error anymore?

Reply:
I do get the following warning:
> WARNING: Found a conflicting export condition for "./package.json". The "default" condition would be overridden by ng-packagr. Please unset it.

But if I don't add this export here the build fails using `nodeLinker: node-modules`

Reply:
Changed `default` export to `node` export